src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -0,0 +1,256 @@
+                onButtonPress: openQBOsync,
+                ...connectionIconSize,
+            },
+            {
+                translationKey: 'workspace.accounting.xero',
+                icon: Expensicons.XeroRound,
+                interactive: false,
+                disabled: true,
+                onButtonPress: () => {},
+                ...connectionIconSize,
+            },
+        ],
+        [openQBOsync, connectionIconSize],
+    );
+
+    const qboConnectionOptionsMenuItems: WorkspaceMenuItem[] = useMemo(
+        () => [
+            {
+                translationKey: 'workspace.accounting.import',
+                icon: Expensicons.Pencil,
+                iconRight: Expensicons.ArrowRight,
+            },
+            {
+                translationKey: 'workspace.accounting.export',
+                icon: Expensicons.Send,
+                iconRight: Expensicons.ArrowRight,
+            },
+            {
+                translationKey: 'workspace.accounting.advanced',
+                icon: Expensicons.Gear,
+                iconRight: Expensicons.ArrowRight,
+            },
+        ],
+        [],
+    );
+
+    const qboConnectionMenuItems: WorkspaceMenuItem[] = useMemo(
+        () => [
+            ...(!isSyncInProgress ? qboConnectionOptionsMenuItems : []),
+            {
+                descriptionTranslationKey: 'workspace.accounting.other',
+                iconRight: Expensicons.DownArrow,
+            },
+        ],
+        [isSyncInProgress, qboConnectionOptionsMenuItems],
+    );
+
+    const menuItems = useMemo(() => {
+        const baseMenuItems = [...(!policyIsConnectedToAccountingSystem ? connectionsMenuItems : []), ...(policyIsConnectedToAccountingSystem ? qboConnectionMenuItems : [])];
+
+        return baseMenuItems.map((item) => ({
+            key: item.translationKey ?? item.descriptionTranslationKey,
+            title: item.translationKey && translate(item.translationKey as TranslationPaths),
+            description: item.descriptionTranslationKey && translate(item.descriptionTranslationKey as TranslationPaths),
+            icon: item.icon,
+            iconRight: item.iconRight,
+            shouldShowRightIcon: !!item.iconRight,
+            interactive: item.interactive,
+            shouldShowRightComponent: !item.iconRight,
+            iconHeight: item.iconHeight,
+            iconWidth: item.iconWidth,
+            disabled: item.disabled,
+            rightComponent: (
+                <Button
+                    onPress={item.onButtonPress}
+                    style={[styles.pl2]}
+                    text={translate('workspace.accounting.setup')}
+                    small
+                    isDisabled={item.disabled}
+                />
+            ),
+            wrapperStyle: [styles.sectionMenuItemTopDescription],
+        }));
+    }, [translate, styles, policyIsConnectedToAccountingSystem, connectionsMenuItems, qboConnectionMenuItems]);
+
+    const threeDotsMenuItems = [
```

Comment:
```suggestion
    const threeDotsMenuItems: ThreeDotsMenuItem[] = [
```

Reply:
I think type ThreeDotsMenuItem is not correct one.
HeaderWithBackButton has not correct type for threeDotsMenuItems. It should be PopoverMenuItem[] instead of ThreeDotsMenuItem[].
I think here is a first case - when we use right icon for ThreeDotsMenuItem. And ThreeDotsMenuItem type does not support it.
But if we check `HeaderWithBackButton` component:
` {shouldShowThreeDotsButton && (
   <ThreeDotsMenu menuItems={threeDotsMenuItems} ...rest />
)}
`
here we see that we pass `threeDotsMenuItems` to `menuItems` - and `menuItems` inside ThreeDotsMenu has type `PopoverMenuItem`